payment/src/main/java/com/kyriba/conference/payment/domain/dto/PaymentDto.java

```diff
@@ -0,0 +1,35 @@
+package com.kyriba.conference.payment.domain.dto;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.kyriba.conference.payment.domain.PaymentMethodType;
+import com.kyriba.conference.payment.domain.PaymentStatus;
+import com.kyriba.conference.payment.domain.vo.Amount;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+/**
+ * @author Igor Lizura
+ */
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class PaymentDto {
+    private long userId;
+
+    @NonNull
```

Comment:
Validation is not applied at controller level.

Reply:
done